backends/jgroups/src/test/java/org/hibernate/search/backend/jgroups/impl/SyncJGroupsBackendTest.java

```diff
@@ -114,10 +114,15 @@ public void testSynchAsConfigured() {
 		}
 		catch (SearchException se) {
 			//Expected: we're inducing the RPC into NPE
-			Throwable cause = se.getCause().getCause().getCause();
-			Assert.assertTrue( "Cause was not a NullPointerException but a " + cause, cause instanceof NullPointerException );
-			Assert.assertEquals( "Simulated Failure", cause.getMessage() );
-			npeTriggered = true;
+			try {
```

Comment:
Yea, I know were you're coming from with this fix.. but I think that @gunnarmorling was suggesting that such improvements should live in a separate commit at least, sometimes in their own JIRA.

For example in this case I don't think this is the best improvement for that NPE but asking to improve on that would delay integration of the other improvements in this PR too.